twake/backend/node/src/services/channels/services/channel/activities/index.ts

```diff
@@ -0,0 +1,3 @@
+import Activities from "./service";
+
+export const getService = (): Activities => new Activities().init();
```

Comment:
You should not init when you get the service. You can implement the `Initializable` interface in the Activities class and then call it in the init block of the main service

Reply:
Done !